portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase

```diff
@@ -520,6 +878,88 @@ definition {
 		}
 	}
 
+	@description = "LPS-185990 Confirm that when creating a filter within the view, there is a search bar field to preselect values"
+	@priority = 5
+	test CanAssertTheSearchBarField {
+		task ("Given the creation of a new data set") {
+			DataSetAdmin.goToDataSetAdminPage();
+
+			DataSetAdmin.createDataSetEntryViaAPI(
+				dataSetName = "Data Set Test",
+				restApplication = "/data-set-manager/fields",
+				restEndpoint = "/",
+				restSchema = "FDSField");
+
+			DataSetAdmin.createFDSViewViaAPI(
+				dataSetName = "Data Set Test",
+				dataSetViewDescription = "FDSViewDescription",
+				key_dataSetViewNameList = "View Test");
+
+			DataSetAdmin.goToSpecificViewPage(
+				dataSetName = "Data Set Test",
+				key_viewName = "View Test");
+		}
+
+		task ("And creating a picklist and Items bike,car,bus") {
+			Picklist.addPicklistViaAPI(picklistName = "Transport");
+
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "bike",
+				itemName = "bike",
+				picklistName = "Transport");
+
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "car",
+				itemName = "car",
+				picklistName = "Transport");
+
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "bus",
+				itemName = "bus",
+				picklistName = "Transport");
+		}
+
+		task ("When the user creates a new filter //Name: Filter test //Filter by: creator //Source option: Transport the selection is multiple") {
+			DataSetAdmin.goToTab(tabName = "Filters");
+
+			LexiconEntry.gotoAdd();
+
+			Click(
+				key_fieldName = "Selection",
+				locator1 = "FormViewBuilder#OBJECT_FIELD_BUTTON_DROPDOWN");
+
+			Type(
+				key_text = "Name",
+				locator1 = "TextInput#ANY",
+				value1 = "Filter test");
+
+			Click(
+				key_class = "btn-secondary",
+				key_index = 3,
+				locator1 = "Button#ANY_WITH_CLASS_AND_INDEX");
+
+			SelectFrameTop();
+
+			Click.javaScriptClick(
+				key_sortOption = "creator",
+				locator1 = "ObjectCustomViews#SORT_OPTION");
+
+			Select(
+				locator1 = "DataSet#SOURCE_OPTIONS_SELECT",
+				value1 = "Transport");
+
+			Click.clickAtNotVisible(
+				key_radioOption = "Single",
+				locator1 = "CommerceEntry#RADIO_BUTTON");
+		}
+
+		task ("Then there is a search in the Preselected Values dropdown") {
+			AssertElementPresent(
+				key_fieldName = "Select a default value for your filter",
+				locator1 = "ObjectCustomViews#FROM_DATE_INPUT");
```

Comment:
Why if you are asserting the search bar you are using ObjectCustomViews#FROM_DATE_INPUT? How is this related to the search bar?